tests/test_admin_actions.py

```diff
@@ -650,7 +650,7 @@ def test_signal_when_published(self):
     @unittest.skip("Skip until collection status bugs fixed")
     @mock.patch("django.contrib.messages.success")
     def test_publish_selected_sets_collection_to_archived_if_all_requests_published(self, messages_mock):
-        # Make sure both moderation requests have been approved
+        # Won't work because the approved_view sets the ARCHIVED state prior to hwo this test is setup
```

Comment:
Spelling mistake

Reply:
Please try and use suggestions for minor changes like this. Makes it much easier to see what the issue actually is.

```suggestion
        # Won't work because the approved_view sets the ARCHIVED state prior to how this test is setup
```